tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -0,0 +1,382 @@
+# -*- encoding: utf-8 -*-
+"""Usage::
+
+    hammer compute_resource [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters::
+
+    SUBCOMMAND                    subcommand
+    [ARG] ...                     subcommand arguments
+
+Subcommands::
+
+    create                        Create a compute resource.
+    info                          Show an compute resource.
+    list                          List all compute resources.
+    update                        Update a compute resource.
+    delete                        Delete a compute resource.
+    image                         View and manage compute resource's images
+
+
+:Requirement: Computeresource
+
+:CaseLevel: Acceptance
+
+:CaseComponent: CLI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import (
+    run_only_on,
+    tier1,
+    tier2,
+    tier3,
+    stubbed,
+    skip_if_bug_open
+)
+from robottelo.test import CLITestCase
+
+
+class ComputeResourceTestCase(CLITestCase):
+    """ComputeResource CLI tests."""
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_create_rhev_with_valid_name(self):
+        """Create Compute Resource of type Rhev with valid name
+
+        :id: 92a577db-144e-4761-a52e-e83887464986
+
+        :steps:
+
+            1. Create a Compute resource of type rhev with valid name.
+
+        :expectedresults: Compute resource is created
+
+        :CaseAutomation: notautomated
+
+        :CaseImportance: Critical
+        """
+
+    @tier1
+    @run_only_on('sat')
+    @stubbed()
+    def test_positive_rhev_info(self):
+        """List the info of RHEV Compute Resource
+
+        :id: 1b18f6e8-c431-41ab-ae49-a2bbb74712f2
+
+        :steps:
+
+            1. Display the information of RHEV compute resource.
```

Comment:
same here. Steps section is needed when you actually perform steps to get to final verification